torch/nn/quantized/modules/conv.py

```diff
@@ -106,7 +106,11 @@ def set_weight_bias(self, w, b):
         # type: (torch.Tensor, Optional[torch.Tensor]) -> None
         self._packed_params = torch.ops.quantized.conv_prepack(
             w, b, self.stride, self.padding, self.dilation, self.groups)
-        self.weight_scale = w.q_scale()
+        if w.qscheme() in set([torch.per_channel_affine, torch.per_channel_symmetric]):
```

Comment:
maybe wrap this check into a function? I think we might need this check somewhere else as well